plugin-template

```diff
@@ -270,6 +293,26 @@ def main():
         else:
             return 2
 
+    valid_release_behavior = [None, "append-version", "replace-previous-version"]
+    if config["ci_update_release_behavior"] not in valid_release_behavior:
+        valid_release_behavior[0] = "null"
+        print("ci_update_release_behavior is not of a valid value: ", valid_release_behavior)
+        return 2
+
+    if new_version_branch := args.add_version_ci_update_branches:
+        write_new_config = True
+        if args.replace_previous_version_ci_update_branches:
+            major_v, minor_v = new_version_branch.split(".", maxsplit=1)
+            minor_v = int(minor_v) - 1
+            try:
+                index = config["ci_update_branches"].index(f"{major_v}.{minor_v}")
+            except ValueError:
+                config["ci_update_branches"].append(new_version_branch)
+            else:
+                config["ci_update_branches"][index] = new_version_branch
+        else:
+            config["ci_update_branches"].append(new_version_branch)
```

Comment:
So we are assuming here, that a new major version will never replace the last version.
This sounds fine to me.

Reply:
Replacing only works for 2.1 -> 2.2. If we released 2.1 -> 3.0 we would still keep 2.1 in the list. And that's probably fine because OTOH it's very rare, OTOH it's probably wise to support the last y of the previous x for a bit longer.